david_laster/class2/compare.rb

```diff
@@ -0,0 +1,7 @@
+puts "enter an integer"
+num1 = gets.chomp.to_i
+if num1 % 2 == 0
+	puts "even"
+else num1 % 2 == 1
```

Comment:
Great solution!

You could even leave your "else" conditional blank and the program will implicitly assume that you mean everything else besides the if conditional :-)